versions/3.0.md

```diff
@@ -107,18 +107,18 @@ For example, if a field has an array value, the JSON array representation will b
 }
 ```
 
-While the API is described using JSON, it does not impose a JSON input/output to the API itself.
+While the API is described using JSON, input to and output from the API is not required to be JSON.
 
 All field names in the specification are **case sensitive**.
 
 The schema exposes two types of fields: Fixed fields, which have a declared name, and Patterned fields, which declare a regex pattern for the field name.
 
-Patterned fields can have multiple occurrences as long as each has a unique name. 
+Multiple occurrences of patterned fields need to have unique names. 
```

Comment:
I suggest using MUST and limiting the scope of the restriction. (Since line 114 uses capital Patterned fields, this should also, but my suggestion moves it to the beginning of the sentence.)

Patterned fields MUST have unique names within the containing object.